### PR TITLE
Final eval blog update for Kush Kothari

### PR DESCRIPTION
**Pull request by kkothari2001**
cc @jpivarski for review

### CODE REVIEW

File: _gsocblogs/2022/blog_IRIS-HEP_KushKothari.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 project: IRIS-HEP
 title: "Uproot + Dask: Blog 1"
 author: Kush Kothari
-date: 25.07.2022
+date: 09.09.2022
```
Comment (jpivarski) [inline _gsocblogs/2022/blog_IRIS-HEP_KushKothari.md:5]: I'm not sure if this is supposed to be a single page, merging the first half and the second, but if it is, then then "Blog 1" name should probably just be "Blog".

**Review by jpivarski — APPROVED** [review]
This is good! It covers all of the great work you've been doing in the second half of your project.

File: _gsocblogs/2022/blog_IRIS-HEP_KushKothari.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 project: IRIS-HEP
 title: "Uproot + Dask: Blog 1"
```
Comment (agheata) [inline _gsocblogs/2022/blog_IRIS-HEP_KushKothari.md:3]: Please consider the recommendations for the final blogs sent in this [message](https://groups.google.com/g/cern-hsf-gsoc-students-2022/c/w_fIfhDdcG8/m/Av5ceK0VCAAJ). The title should rather match your project name. For the intro, you could just use the first two sentences from your actual introduction. If no photo, just use your GitHub avatar link. All this is NOT mandatory, if you chose not to do it, just mark this as resolved.

Comment (kkothari2001) [reply _gsocblogs/2022/blog_IRIS-HEP_KushKothari.md]: Made the suggested changes. Thank you

**Review by agheata — APPROVED** [review]

**Pull request merged**